app/src/main/res/layout/activity_event_edit_management.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".view.activity.admin.EventEditManagementActivity">
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/id_recycler_event_edits"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
```

Comment:
Probably better to set height to 0dp in recycler views

Reply:
Yes!